src/course-updates/update-form/UpdateForm.test.jsx

```diff
@@ -48,18 +52,32 @@ const courseUpdatesInitialValues = (requestType) => {
 };
 
 const renderComponent = ({ requestType }) => render(
-  <IntlProvider locale="en">
```

Comment:
Are these course-updates changes necessary?

Reply:
yes, because of the changes to the editor props. The editors for the course updates page use the same Tiny MCE editors

Reply:
gotcha, ty